src/engines/quickjs.js

```diff
@@ -29,6 +31,9 @@ class QuickJSInstaller extends Installer {
   }
 
   static resolveVersion(version) {
+    if (['darwin-arm64', 'darwin-x64', 'linux-arm64'].includes(platform)) {
+      return '6.0.0';
```

Comment:
you can make a request to the github api to get the current release of the `napi-bindings/quickjs-build` repo. other engines here do the same if you wanna just copy the code over.

Reply:
fixed